html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -13,59 +13,59 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <header>
+      <h1>Jane Doe</h1>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="#">About Me</a>
+        <a href="#">Contact</a>
+        <a href="#">Skillset</a>
+      </nav>
+    </header>
     <!--Looks like the following part is the central content of our webpage.-->
-    <div>
-      <div id="about-me">
-        <div>About me</div>
-        <div>
+    <main>
+      <section id="about-me">
+        <h2>About me</h2>
+        <p>
           Hi there! Welcome to my personal website. Here is gathered the most
           important information about me. If you want to work with me or get to
           know me, please don't hesitate to get in contact.
-        </div>
+        </p>
         <!--What element could be used to combine an image with a descriptive caption?-->
-        <div>
+        <article>
```

Comment:
Avoid using article tag when there is no header present, maybe a section tag instead.